#!/usr/bin/env python

# steembase.storage.configStorage has error
from steem.cli import configStorage

steemd_nodes = [
    'https://gtg.steem.house:8090',
    'https://rpc.esteem.ws',
    'https://rpc.steemviz.com',
]

configStorage['nodes'] = ','.join(steemd_nodes)
